src/components/forms/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback } from 'react'
+import { View, TouchableWithoutFeedback } from 'react-native'
+import { useStyles } from '../../../theme'
+import { Box } from '../../structure'
+import { BodyText } from '../../text'
+import { shameStyles } from '../../../theme/shame-styles'
+
+export interface RadioButtonProps {
+  /**
+   * Label to display next to the Radio Button.
+   */
+  label: string
+  /**
+   * True if selected.
+   */
+  checked: boolean
+  /**
+   * Called when selection state changes. Should propagate change to `checked` prop.
+   */
+  onChange: (checked: boolean) => void
```

Comment:
Maybe move the event handlers at the bottom so structural and presentational properties stay together

1. Structural: `label`, `helpText`
2. State: `checked`
3. Handlers: `onChange`

Reply:
Done :wink: